app/src/main/res/values/strings.xml

```diff
@@ -673,41 +673,41 @@
     <string name="dsa_affine_y">Affine y coordinate</string>
     <string name="critical_exts">Critical Extensions</string>
     <string name="non_critical_exts">Non-critical Extensions</string>
-    <string name="usage_access_not_supported">Could not request usage access as the corresponding settings page does not exist.</string>
+    <string name="usage_access_not_supported">Could not request usage access, as the corresponding settings page does not exist.</string>
     <string name="packages">packages</string>
     <string name="onboard">Onboard</string>
     <string name="do_you_want_to_uninstall_and_install">Do you want to uninstall and install the app again?</string>
-    <string name="signature_mismatch_for_system_apps">Could not install app because the installed app is a system app with different signature.</string>
+    <string name="signature_mismatch_for_system_apps">Could not install app because the installed one is a system app with different signature.</string>
```

Comment:
I think this needs some rework, something like _Could not install app because a system app has this package name with a different signature._